src/xdocs/config_javadoc.xml

```diff
@@ -814,6 +736,47 @@ public int checkReturnTag(final int aTagIndex,
   &lt;property name="excludeScope" value="protected"/&gt;
 &lt;/module&gt;
         </source>
+
+        <p>
+          To configure the check to validate <code>throws</code> tags, you can use following config.
+          ATTENTION: checkstyle do not have information about hierarchy of exception types so usage
+          of base classes will be considered as separate exception type. As workaround you need to
+          specify both types base and exact type in javadoc.
+        </p>
+        <source>
+&lt;module name="JavadocMethod"&gt;
+  &lt;property name="validateThrows" value="true"/&gt;
+&lt;/module&gt;
+        </source>
+        <source>
+/**
+ * Actually throwed is child class of class that is declared in throws.
+ * It is limitation of checkstyle (as we do not know type hierarchy).
+ * Code and javadoc is valid BUT checkstyle can not distinguish it.
+ * @param file some file
+ * @throws IOException if some problem
+ */
+public void doSomething8(File file) throws IOException {
+    if (file == null) {
+        throw new FileNotFoundException(); // violation
+    }
+}
+
+/**
+ * Special throw type referencing in javadoc even first is parent of second type
```

Comment:
Instead of `Special` wouldn't it make more sense to say `Exact`? Also this line is missing a period on the end.

Reply:
done